index.js

```diff
@@ -0,0 +1,2 @@
+const nombreUsuario = prompt('ingrese su nombre');
+alert(`hola ${nombreUsuario}`);
```

Comment:
creo que no seria indicado hacer interpolacion de cadenas
```suggestion
alert(`hola + nombreUsuario`);
```